src/navigations/root-stack.js

```diff
@@ -95,31 +95,11 @@ export const RootNavigator = ({currentScreen}) => {
     };
   }, []);
 
-  const isUnauthenticated = initialStartup.id === null || initialStartup.id === '';
-
-  const getPaddingTop = (screenName) => {
-    'worklet';
-
-    if (isUnauthenticated || ['TopicPageScreen', 'TopicMemberScreen'].includes(screenName)) {
-      return 0;
-    }
-    return insets.top;
-  };
-
-  const getInsetTopColor = () => {
-    'worklet';
-
-    return currentScreen === 'SampleChatScreen' ? colors.anon_primary : colors.white;
-  };
-
   return (
     <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
       <View
         style={{
-          height: '100%',
-          paddingBottom: isUnauthenticated ? 0 : insets.bottom,
-          paddingTop: getPaddingTop(currentScreen),
-          backgroundColor: getInsetTopColor()
+          height: '100%'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the dynamic padding and background color that were previously applied to the View component. This could potentially affect the layout and appearance of your application, especially on screens where these properties are important. If this change is intentional and you've verified that it doesn't negatively impact your UI, then it's fine. Otherwise, you might want to reconsider this change.

```diff
-  const getPaddingTop = (screenName) => {
-    'worklet';
-
-    if (isUnauthenticated || ['TopicPageScreen', 'TopicMemberScreen'].includes(screenName)) {
-      return 0;
-    }
-    return insets.top;
-  };
-
-  const getInsetTopColor = () => {
-    'worklet';
-
-    return currentScreen === 'SampleChatScreen' ? colors.anon_primary : colors.white;
-  };
-
-  return (
-    <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
-      <View
-        style={{
-          height: '100%',
-          paddingBottom: isUnauthenticated ? 0 : insets.bottom,
-          paddingTop: getPaddingTop(currentScreen),
-          backgroundColor: getInsetTopColor()
-        }}>
-        <NetworkStatusIndicator hide={true} />
+  return (
+    <LoadingStartupContext.Provider value={loadingStartup.loadingUser}>
+      <View style={{height: '100%'}}>
+        <NetworkStatusIndicator hide={true} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->